plugins/systemlogs/api/api.js

```diff
@@ -240,14 +245,14 @@ var plugin = {},
                         }
 
                         compareChangesInside(dataafter[keys[i]], databefore[keys[i]], before[keys[i]], after[keys[i]]);
-                        if (typeof dataafter[keys[i]] === "object" && typeof databefore[keys[i]] === "object" && Object.keys(dataafter[keys[i]]) == 0 && Object.keys(databefore[keys[i]]) == 0) {
+                        if (typeof dataafter[keys[i]] === "object" && typeof databefore[keys[i]] === "object" && Object.keys(parseInt(dataafter[keys[i]])) === 0 && Object.keys(parseInt(databefore[keys[i]])) === 0) {
```

Comment:
This seems weird, we are getting keys of sub object, but suddenly, we convert it to number?
`Object.keys(parseInt(dataafter[keys[i]]))`